### PR TITLE
Some minor updates to JetBrains IDE page

### PR DESCRIPTION
**Pull request by parlough**
- WebStorm no longer bundles the Dart plugin by default but we should still mention it, as it is more lightweight than IntelliJ Ultimate while supporting web functionality.
- Link to Android Studio install
- Remove Eclipse migration link. We're long past the days of Eclipse and Dart Editor support.

### CODE REVIEW

File: src/tools/jetbrains-plugin.md [hunk 1]
```diff
@@ -4,22 +4,18 @@ description: Use Dart with a variety of IDEs and editors from JetBrains.
...
 
 The [Dart plugin][] adds Dart support to JetBrains IDEs such as
-IntelliJ IDEA and Android Studio.
+IntelliJ IDEA, WebStorm, and Android Studio.
 IntelliJ IDEA is an intelligent Java IDE
 with support for many other languages and frameworks.
+WebStorm is an IDE based on IntelliJ IDEA
+that's focused on web development and debugging.
 Android Studio is an IDE based on IntelliJ IDEA
 that's used for Android and Flutter development.
```
Comment (atsansone) [inline src/tools/jetbrains-plugin.md:13]: **issue:** Limit the marketing speak for JetBrains. Use active voice where possible.

**suggestion:** Link to the apps, if you want.

```suggestion
The [Dart plugin][] adds Dart support to nine of the IDEs developed by JetBrains s.r.o. 
These IDEs provide features unique to specific programming languages.
The IDEs recommended for Flutter development include:
- IntelliJ IDEA specializes in JVM-based language development.
- WebStorm specializes in JavaScript development.
- Android Studio specializes in Android and Flutter development.
```

**Review by atsansone — CHANGES_REQUESTED** [review]
@parlough : I would tighten up some of the language, removing some of the sales language from the IDE descriptions.

File: src/tools/jetbrains-plugin.md (hunk 1 above)
Comment (parlough) [reply src/tools/jetbrains-plugin.md]: Thanks. I've applied the suggestions with modifications.

**Review by atsansone — APPROVED** [review]
LGTM.

**Pull request merged**